enclave-runtime/Cargo.toml

```diff
@@ -72,7 +75,8 @@ webpki = { git = "https://github.com/mesalock-linux/webpki", branch = "mesalock_
 webpki-roots = { git = "https://github.com/mesalock-linux/webpki-roots", branch = "mesalock_sgx" }
 log = { git = "https://github.com/integritee-network/log-sgx" }
 env_logger = { git = "https://github.com/integritee-network/env_logger-sgx" }
-serde = { tag = "sgx_1.1.3", git = "https://github.com/mesalock-linux/serde-sgx" }
+serde = { tag = "sgx_1.1.3", git = "https://github.com/mesalock-linux/serde-sgx", features = ["alloc", "mesalock_sgx"] }
```

Comment:
Hm. Is that tag needed?

Reply:
I don't know, it was there before, so I kept it?

Reply:
I'm asking because of the `serde_derive` below. This didn't have it and I guess this is where the extra Cargo.lock import now comes from. Nothing major, but if you have time, can you try to remove the extra Cargo.lock import?

Reply:
I will revert my change again, that should also remove that extra `serde_derive` import.